spring-reactive-modules/spring-reactive-data/src/main/java/com/baeldung/r2dbc/configuration/R2DBCConfiguration.java

```diff
@@ -2,6 +2,10 @@
 
 import io.r2dbc.h2.H2ConnectionConfiguration;
 import io.r2dbc.h2.H2ConnectionFactory;
+import io.r2dbc.spi.Connection;
```

Comment:
Do we need these imports?

Reply:
Thanks @Maiklins, we don't need it. Removed